config/puma.rb

```diff
@@ -0,0 +1,9 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 2)
+threads_count = Integer(ENV['MAX_THREADS'] || 5)
```

Comment:
Will Heroku set those to match my current plan? The [documentation](https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server) is pretty hazy on that front.